src/client/SplitPaneResizeContext.tsx

```diff
@@ -80,18 +80,21 @@ export const SplitPaneResizeContext =
 
 type SplitPaneState = {
   // The width of the sidebar, in pixels.
-  sidebarWidth: number;
-  codeEditorWidth: number;
+  leftPanelWidth: number;
+  rightPanelWidth: number;
   isDraggingLeft: boolean;
   isDraggingRight: boolean;
 
   // False on first render
   isInitialized: boolean;
+  // whether the last change to the state was performed manually by the user
+  // false in case of wimdow resizes
+  manual: boolean;
```

Comment:
This change is unrelated to touch support. It would be best to chunk this as a separate PR.